src/map/packet_system.cpp

```diff
@@ -2289,6 +2291,24 @@ void SmallPacket0x04B(map_session_data_t* const PSession, CCharEntity* const PCh
         {
             login_message = login_message + (settings::get<std::string>("main.TRUST_ALTER_EGO_EXPO_MESSAGE"));
         }
+        if (settings::get<bool>("main.VANAVERSARY_ANNOUNCE"))
```

Comment:
This logic all lives in Lua now in server.lua, this'll need to be rebased and then changed to account for that 👍 

https://github.com/LandSandBoat/server/pull/4132

Reply:
Thanks! A lot has changed in the last two months. Converting back to draft while I do some reworking to address this and a few other items.